lms/djangoapps/dashboard/decisions/0002-deprecate-sysadmin-dashboard-adr

```diff
@@ -0,0 +1,45 @@
+1. Deprecating the Sysadmin Dashboard
+---------------------
+
+Status
+------
+
+Draft
+
+Context
+-------
+
+Maintaining the sysadmin dashboard is challenging, and it is not widely used. The code is part of the lms
+application, even though most of it's use cases are relevant to course authoring.
+
+The sysadmin dashboard would be better suited as a pluggable django application, using appropriate APIs in the
+cms application
+
+Decision
+--------
+
+In order to deprecate the sysadmin dashboard and move it to a pluggable django application, the followings APIs
+would need to be added and/or moved into the cms application
+
+1. Create a cms user account
+
+  https://github.com/mitodl/edx-platform/blob/50dd1238408dc6785f022d8540961f96e0d6bb4f/lms/djangoapps/dashboard/sysadmin.py#L113-L151
+
+2. Delete a cms user.
```

Comment:
Should we create a separate list for features that could be removed/replaced? rather than including it in this list of APIs needed?

Reply:
sure